pkg/kv/kvprober/kvprober.go

```diff
@@ -335,9 +368,21 @@ func (p *Prober) Start(ctx context.Context, stopper *stop.Stopper) error {
 					return
 				}
 
-				probeCtx, sp := tracing.EnsureChildSpan(ctx, p.tracer, opName+" - probe")
-				probe(probeCtx, pl)
-				sp.Finish()
+				func() {
```

Comment:
I think we should introduce a cluster setting in below file called `kvprober.tracing.enabled` that controls whether `kvprober` takes traces at all. If we backport this, we will set it to `false` for safety. If we just release it in a new major version, we will set it to `true`.

https://github.com/cockroachdb/cockroach/blob/master/pkg/kv/kvprober/settings.go